scripts/calculator.js

```diff
@@ -1,3 +1,27 @@
-document.getElementById("button3").onclick = function() {
```

Comment:
I know I removed it, but this 'onclick' event listener is actually the right idea.

In fact, we know we're going to need to set up a bunch of these for the different buttons on the calculator. Why don't you create a single function that handles setting them all up? Something like "initializeListeners".